.github/workflows/buildManylinux.yml

```diff
@@ -0,0 +1,91 @@
+name: Manylinux Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      release_id:
+        description: 'Release id to upload artifacts to'
+        default: ''
+      python_package_version:
+        description: 'Version to use for creating the Python package'
+        default: ''
+
+jobs:
+  build:
```

Comment:
what is this doing that makes it "manylinux"? It looks the same as the other one.

Also, let's not do this on every pull request / push at least if it is just for testing / ondemand.

Reply:
not yet hooked up. This just sets up the oneshot so we can trigger and iterate with the dummy buildManylinux.yml action. The idea would be only push on release and when it gets called from the scheduled workflow.